.github/workflows/ci.yml

```diff
@@ -69,13 +69,12 @@ jobs:
 
       - name: Run Tests
         env:
-          TFE_TOKEN: "${{ env.TFE_TOKEN }}"
```

Comment:
I just found out that if this were fetched from the github secrets config, it would output `***` in the action log. Just FYI for the future